i18n/zh/docusaurus-plugin-content-docs/version-v1.0/best-practices/acronjob+broadcastjob.md

```diff
@@ -0,0 +1,232 @@
+---
+title: BroadcastJob + Advanced CronJob 定期清理节点磁盘
+---
+相较于 CPU、内存这类具有较高灵活性的资源，节点磁盘存储可用空间在 Kubernetes 原生体系中基本上处于一种单调递减的宿命，磁盘压力检测和控制成为节点运维必不可少的一环，否则过大的磁盘压力可能导致该节点无法调度，甚至导致节点上的 Pod 被驱逐等等一系列副作用，影响集群的稳定性。
+
+Kubernetes Job 的模式显然非常合适做这类临时性的磁盘清理工作，因为它不像常驻宿主机的 Agent 需要消耗节点资源，Job 只需要临时地占用一些节点资源，任务执行完毕后会自动释放。
+
+OpenKruise 就专门针对这类需求打造了 BroadcastJob 和 Advanced CronJob。其中，BroadcastJob 允许用户以类似 DaemonSet 调度的方式来编排 Pod，当用户创建 BroadcastJob 后，它会默认在集群的每一个 worker 节点创建一个 Pod，执行完成之后会自动对 Pod 进行清理。同时，搭配 Advanced CronJob，可以将 BroadcastJob 进行定时发下，从而实现定时清理节点磁盘的能力。本文会以定期清理节点中存储的无用镜像这一场景来进行演示如何使用 Advanced CronJob 和 BroadcastJob。
```

Comment:
It is suggested that some comparisons can be made here between BroadcastJob and k8s job, and Advanced CronJob and k8s cron job. Mainly reflects why the native k8s job is not quite in line, so choose kruise.